### PR TITLE
Improve Outflux Tutorial content formatting

### PR DESCRIPTION
**Pull request by blagojts**

### CODE REVIEW

File: tutorials/outflux.md [hunk 1]
```diff
@@ -62,31 +65,37 @@ One of Outflux’s features is the ability to discover the schema of an InfluxDB
 We can now create a TimescaleDB hypertable ready to receive the demo data we inserted into the InfluxDB instance. If you followed the tutorial and inserted the data from the example, there should be a `taxi` measurement in the `outflux_tutorial` database in the InfluxDB instance.
 
 The `schema-transfer` command of Outflux can work with 4 schema strategies:
-* `ValidateOnly`: checks if a specified TimescaleDB database has a hypertable with the proper columns and if it’s partitioned properly, but will not perform modifications
-* `CreateIfMissing`: validates everything if it exists, and it will check if the TimescaleDB  extension is installed (if the hypertable is missing, it will create it  and partition it properly)
+* `ValidateOnly`: checks if a the TimescaleDB extension is installed, a specified database has a hypertable with the proper columns, and if it’s partitioned properly, but will not perform modifications
```
Comment (mfreed) [inline tutorials/outflux.md]: "a the" -> the

File: tutorials/outflux.md [hunk 2]
```diff
@@ -109,24 +118,30 @@ CREATE EXTENSION IF NOT EXISTS timescaledb
...
-> --schema-strategy=DropAndCreate
+ --input-server=http://localhost:8086 \
+ --output-conn="dbname=postgres user=postgres" \
+ --schema-strategy=DropAndCreate
 ```
 
-Here we’re using the DropAndCreate strategy that will drop any previous table named “cpu” and create it before piping the data. The migrate command supports several flags that offer the user flexibility in the selection of data to be migrated. One of them is the --limit flag that will only export the first N rows from the InfluxDB database ordered by time. The output of the migrate command with a N=10 limit should look like this:
+Here we’re using the DropAndCreate strategy that will drop any previous table named “cpu” and create it before piping the data. 
+The migrate command supports several flags that offer the user flexibility in the selection of data to be migrated. 
+One of them is the --limit flag that will only export the first N rows from the InfluxDB database ordered by time.
```
Comment (mfreed) [inline tutorials/outflux.md]: should this be `--limit` ?

File: tutorials/outflux.md [hunk 3]
```diff
@@ -52,7 +54,8 @@ $ influx -import -path=outflux_taxit.txt -database=outflux_tutorial
 2019/03/27 11:39:11 Failed 0 inserts
 ```
 
-The data in the file is without a timestamp so the current time of the Influx server will be used at the time of insert. All the data points belong to one measurement `taxi`. The points are tagged with location, rating and vendor. Four fields are recorded: fare, mta_tax, tip and tolls.
+The data in the file is without a timestamp so the current time of the Influx server will be used at the time of insert. 
+All the data points belong to one measurement `taxi`. The points are tagged with location, rating and vendor. Four fields are recorded: fare, mta_tax, tip and tolls.
```
Comment (mfreed) [inline tutorials/outflux.md]: Nit:  Oxford comma for consistency:  "location, rating, and vendor"   "fare, mta_tax, tip, and tolls."

File: tutorials/outflux.md [hunk 4]
```diff
@@ -109,24 +118,30 @@ CREATE EXTENSION IF NOT EXISTS timescaledb
 
 ## Data Migration [](migration)
 
-Schema transfer is useful, but it’s not what we built Outflux for. You can do schema-transfer and data migration in one with the `migrate` command. The connection options available are the same (and you can check them out on the [public repo][outflux-connection]). You can transfer a complete InfluxDB database with each measurement being exported as a separate table, or you can select which measurements to export.
+Schema transfer is useful, but it’s not what we built Outflux for. 
+You can do schema-transfer and data migration in one with the `migrate` command.
```
Comment (solugebefola) [inline tutorials/outflux.md]: maybe emphasize _and_?

File: tutorials/outflux.md (hunk 2 above)
Comment (solugebefola) [reply tutorials/outflux.md]: Do you mean "--limit" in backticks?

**Pull request merged**